docs/howto/account-billing/forgot-your-password.md

```diff
@@ -0,0 +1,55 @@
+# Resetting your password or reclaiming your username
+
+Via the {{gui}}, you can reset a lost password or reclaim your username.
+
+## Reset lost password
+
+Begin by navigating to the https://{{gui_domain}}. At the bottom left-hand side of the page, click on the Forgot your password? button.
+
+![Picture with a orange button with white text Forgot your password? framed with red color](assets/Forgot-your-password.png)
+
+A pop-up window titled Reclaim lost password!, appears. Type in your username, then click on the Reclaim password! button.
+
+![Picture with a field Enter your username and a orange button with white text Reclaim password! framed with red color](assets/Reclaim-lost-password.png)
+
+The content of the pop-up window changes, informing you that an email has been sent to you with instructions on how to reset your password.
+
+Click the Ok! button, then go ahead and check your inbox (or SPAM/junk folder) for an email with the subject **_Cleura Account password reset request_**.
+
+![Picture with text information An email has been sent to you with instructions on how to reset your password and a orange button with white text Ok! framed with red color](assets/OK-reclaim-lost-password.png)
+
+The email will look like the one shown below. Just click the link to create a new password.
+
+![Picture with information to follow the link and create a new password](assets/Reset-password-link.png)
+
+After clicking the link you are landed on a new page, where you are asked to set a new password.
+
+![Picture how you fill in your new password and a orange button with white text Set new password framed with red color](assets/Choose-new-password.png)
+
+Come up with a strong password and type it in twice, for security reasons. Click the "Set new password" button when you are done. You will be informed that the password has been reset, so click the _Go back to login page_ button to login.
+
+![Picture with information that your new password was succesfully changed](assets/Reset-completed.png)
+
+## Reclaim your username
+
+Navigate to the https://{{gui_domain}}, and at the bottom left-hand side of the page click the _Forgot your password?_ button.
```

Comment:
Please replace https://{{gui_domain}} with [{{gui}}](https://{{gui_domain}}), and _Forgot your password?_ with "Forgot your password?".